backend/app/utils/Ocr.scala

```diff
@@ -67,16 +67,28 @@ object Ocr {
         throw new OcrSubprocessCrashedException(exitCode, stderr.getOutput)
     }
   }
-
+  
   // TODO MRB: allow OcrMyPdf to read DPI if set in metadata
   // OCRmyPDF is a wrapper for Tesseract that we use to overlay the OCR as a text layer in the resulting PDF
   def invokeOcrMyPdf(lang: String, inputFilePath: Path, dpi: Option[Int], stderr: OcrStderrLogger, tmpDir: Path): Path = {
     val tempFile = tmpDir.resolve(s"${inputFilePath.getFileName}.ocr.pdf")
-    val cmd = s"ocrmypdf --redo-ocr -l $lang ${dpi.map(dpi => s"--image-dpi $dpi").getOrElse("")} ${inputFilePath.toAbsolutePath} ${tempFile.toAbsolutePath}"
-
     val stdout = mutable.Buffer.empty[String]
-    val process = Process(cmd, cwd = None, extraEnv = "TMPDIR" -> tmpDir.toAbsolutePath.toString)
-    val exitCode = process.!(ProcessLogger(stdout.append(_), stderr.append))
+
+    def process(redoOcr: Boolean): Int = {
```

Comment:
An enum of the options would be a slightly clearer argument to accept here. It would also provide a place to put comments describing what they mean and linking to any docs

Reply:
(Could include `ForceOcr` as well in case we need it for something in future... slight YAGNI violation but given the edge cases for ocr'ing PDFs I think it could help future devs to have the menu of options laid out in the codebase)

Reply:
Good shout - have added this